packages/components/src/globals/grid/__tests__/grid-test.js

```diff
@@ -9,68 +9,86 @@
 
 const { createSassRenderer } = require('@carbon/test-utils/scss');
 
-const render = createSassRenderer(
-  __dirname,
-  `
+const tests = useDartSass => {
+  const { renderer: sassRender } = createSassRenderer(
+    __dirname,
+    '',
+    useDartSass
+  );
+  const render = content =>
+    sassRender(`
 $css--font-face: false;
 $css--helpers: false;
 $css--body: false;
 $css--use-layer: false;
 $css--reset: false;
 $css--plex: false;
-`
-);
+${content}
+`);
+
+  jest.setTimeout(10000);
+
+  let compiler = ' in node-sass';
+  if (useDartSass) {
+    compiler = ' in dart-sass';
+  }
 
-describe('_grid.scss', () => {
-  it('should generate grid code when the grid feature flag is on', async () => {
-    const { result } = await render(`
+  describe(`_grid.scss${compiler}`, () => {
+    it('should generate grid code when the grid feature flag is on', async () => {
+      const { result } = await render(`
+$feature-flags: (grid: true);
 @import '../grid';
 `);
-    expect(result.css.toString()).toMatchSnapshot();
-  });
+      expect(result.css.toString()).toMatchSnapshot();
+    });
 
-  it('should export a 12 column grid by default', async () => {
-    const { result } = await render(`
+    it('should export a 12 column grid by default', async () => {
+      const { result } = await render(`
+$feature-flags: (grid: true);
```

Comment:
Just wanted to double-check if this change is intentional.

Reply:
Ah I must have missed that when I was resolving some conflicts ... fixed!